woocommerce/woocommerce-ios
# Remove "Check for Nested Frameworks" build phase

### Description

This PR addresses the current warnings regarding the WooCommerce target build phase:

> warning build: Run script build phase 'Check for nested frameworks' will be run during every build because it does not specify any outputs. To address this warning, either add output dependencies to the script phase, or configure it to run in every build by unchecking "Based on dependency analysis" in the script phase.

If we do not specify input or output files, Xcode will runs these scripts every time we build the target, regardless of modification status during incremental builds. As this could have a significant impact on compile time, we want Xcode to only run the scripts on the files that actually change.

As this script was added to fix an issue with Xcode 12.4, and is no longer a problem, this PR deals with removing the script and build phase that runs it.

### Changes

- Removed "Check for nested frameworks" build phase
- Deleted `check-nested-frameworks.sh` script

### Testing instructions
- Confirm the app builds successfully, and the phase is no longer part of building the WooCommerce target
- Confirm the warning message for "Check for nested frameworks" has disappeared

<img width="600" alt="Screenshot 2022-11-23 at 12 21 48" src="https://user-images.githubusercontent.com/3812076/203464442-d3fb6b0c-3736-4b61-9fea-4c984d2f86d0.png">

### Screenshots



---
